lib/open_food_network/xero_invoices_report.rb

```diff
@@ -204,7 +204,7 @@ def total_transaction(order)
     end
 
     def tax_on_shipping_s(order)
-      tax_on_shipping = order.all_adjustments.shipping.sum(:included_tax) > 0
+      tax_on_shipping = order.shipments.sum("additional_tax_total + included_tax_total").positive?
```

Comment:
it'd be worth taking a look at this performance improvement once we deploy. It may give us an idea of how much we can expect from things like these :ok_hand: .